sampleApps/Serilog.Sinks.Raygun.SampleConsoleApp/Program.cs

```diff
@@ -17,15 +19,19 @@
         "CustomGroupKeyProperty",
         "CustomTagsProperty",
         "CustomUserInfoProperty",
-        onBeforeSendArguments =>
+        // onBeforeSendArguments =>
+        // {
+        //     Console.Out.WriteLine("OnBeforeSend called with the following arguments: " + onBeforeSendArguments);
+        //     //Updating machine name
+        //     onBeforeSendArguments.RaygunMessage.Details.MachineName = "Serilog.Sinks.Raygun.SampleConsoleApp Machine";
+        //
+        //     //Testing throwing an exception in Action
+        //     throw new Exception("Exception thrown in callback action..");
+        //     
+        // }, 
```

Comment:
Be good to either uncomment this code again, or remove if it's not helpful/relevent

Reply:
I left it in because its part of a sample app